doc/api/fs.markdown

```diff
@@ -576,6 +576,9 @@ These stat objects are instances of `fs.Stat`.
 If you want to be notified when the file was modified, not just accessed
 you need to compare `curr.mtime` and `prev.mtime`.
 
+_Note: when an `fs.watchFile` operation results in an `ENOENT` error, it will
+invoke the callback once. This is a change in functionality since v0.10._
+
```

Comment:
I feel that it would be better if we can have an example here to show how to differentiate between `ENONET` case and the normal case.